src/board_builder/structures/pose_location.py

```diff
@@ -0,0 +1,45 @@
+import datetime
+
+import numpy as np
+from scipy.spatial.transform import Rotation as R
+
+from src.common.structures import Matrix4x4, Pose
+
+
+class PoseLocation:
+    def __init__(self, object_id):
+        self._id = object_id
+        self._timestamp = str(datetime.datetime.now())
+
+        self.__TMatrix = np.eye(4)
+        self.__RMAT_list = []  # Rotation matrix
+        self.__TVEC_list = []  # Translation vector
```

Comment:
Why leading double underscore? This triggers a special behaviour in Python called name mangling.